### ISSUE TITLE
Specify which image add method to use

### ISSUE THREAD

**Issue by liaprins-czi** [napari/product-heuristics-2020#3]
There seem to be four ways to add images based on a user's use case and preferred working style (notebook vs in console?) but unsure if choosing one over another has downstream constraints.

The image example in tutorial also does not work.
__________________________________________________________
**Recommendation**

Update documentation to explain if there’d be a certain reason to choose one over another.

Minor issue
◆◇◇   Low content writing work
○○○   No design work
▽▽▽   No development work

<img width="845" alt="imageadd4ways" src="https://user-images.githubusercontent.com/53277128/95917267-cbaafe00-0d5e-11eb-89ef-e6560f90ef6e.png">

**Comment by goanpeca** [comment]
Is this still the case @napari/core-devs ?

---

This could be something to work on @ppwadhwa

**Comment by GenevieveBuckley** [comment]
> The image example in tutorial also does not work.

Can you elaborate on this @liaprins-czi? Both examples work for me, so I'm curious about what you might be doing differently.

**Comment by sofroniewn** [comment]
yeah, there are a couple different options still. we're trying to reduce the complexity here (removing some of the `gui_qt` context manager). I suspect @GenevieveBuckley that `skimage` might not have been installed at the time if the test was done in a clean environment and that was confusing.

I think if @GenevieveBuckley can confirm that what we have right now still works (which it sounds like it does) and continue to make some of the desired improvement, then I might recommend @ppwadhwa hold off for now.

I could imagine us doing more sustained documentation pushes over the next few months too

**Comment by liaprins-czi** [comment]
Yes, I think it was likely `skimage`-related, when the example didn't work for me. I think if that is a dependency, we should call that out in the documentation, and how to fix it if possible (if still relevant).
